pkg/storage/bloom/v1/builder_test.go

```diff
@@ -548,9 +548,8 @@ func TestMergeBuilder_Roundtrip(t *testing.T) {
 	builder, err := NewBlockBuilder(blockOpts, writer)
 	require.Nil(t, err)
 
-	checksum, _, err := mb.Build(builder)
+	_, _, err = mb.Build(builder)
 	require.Nil(t, err)
-	require.Equal(t, uint32(0x2a6cdba6), checksum)
```

Comment:
Do we need to add checksum checking back to the test later on?

Reply:
I removed it, because the checksum changes every time you change someone on the binary encoding.
When the format is stable again, we can add it back. I will add a comment.